ssofact.module

```diff
@@ -20,3 +22,52 @@ function ssofact_help($route_name, RouteMatchInterface $route_match) {
       return $output;
   }
 }
+
+/**
+ * Implements hook_css_alter().
+ */
+function ssofact_css_alter(&$css) {
+  if (\Drupal::routeMatch()->getRouteName() !== 'ssofact.user') {
+    return;
+  }
+  ssofact_convert_asset_paths($css);
+}
+
+/**
+ * Implements hook_js_alter().
+ */
+function ssofact_js_alter(&$js) {
+  if (\Drupal::routeMatch()->getRouteName() !== 'ssofact.user') {
+    return;
+  }
+  ssofact_convert_asset_paths($js);
+}
+
+/**
+ * Implements hook_file_url_alter().
+ */
+function ssofact_file_url_alter(&$uri) {
+  if (\Drupal::routeMatch()->getRouteName() !== 'ssofact.user') {
+    return;
+  }
+  if (strpos($uri, '/') === 0) {
```

Comment:
Isn't clear to me what are you achieving in this `if` block. Can you add a comment?

Have you check `UrlHelper` (https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Component%21Utility%21UrlHelper.php/class/UrlHelper/8.2.x)? maybe can simplify this code...

Reply:
It's checking for a domain-relative URL path – looks pretty simple/clear to me at least…

A comment to explain that all of this is happening to convert relative asset paths into absolute ones would certainly help though 👍